packages/web-components/src/tabs/tabs.stories.ts

```diff
@@ -0,0 +1,235 @@
+import { html } from '@microsoft/fast-element';
+import { TabsOrientation } from '@microsoft/fast-foundation';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../helpers.stories.js';
+import type { Tabs as FluentTabs } from './tabs.js';
+import './define.js';
+import '../tab/define.js';
+import '../tab-panel/define.js';
+import { TabsAppearance as TabsAppearanceValues, TabsSize } from './tabs.options.js';
+
+type TabsStoryArgs = Args & FluentTabs;
+type TabsStoryMeta = Meta<TabsStoryArgs>;
+
+const defaultTabIds = ['first-tab', 'second-tab', 'third-tab', 'fourth-tab'];
+
+const tabsDefault = html`
+  <fluent-tabs
+    orientation=${component => component.orientation}
+    appearance=${component => component.appearance}
+    ?disabled=${component => component.disabled}
+    size=${component => component.size}
+    activeid=${component => component.activeid}
```

Comment:
I believe that we've been pretty consistently using `x => x.property`, we should probably leverage the same thing just for consistency.

Reply:
I saw that, and I am happy to change this to x => x.property for the sake of consistency. For the record though, in my opinion, it's not the best dev practice to give variables arbitrary names (unless it's a simple data type) and we should consider changing this project wide.

Reply:
If you are particularly passionate about it and want to file an RFC, please do so. I'll note, for the stories, I think it's equally as clear as component. With that said, I don't really have a strong opinion besides the fact that we already use the format for templates and we should maintain that approach over verbosity IMO. For stories, don't really have a preference, but it would be odd to treat them differently.